src/zkevm_specs/evm/instruction.py

```diff
@@ -328,40 +378,40 @@ def mul_word_by_u64(self, multiplicand: RLC, multiplier: Expression) -> Tuple[RL
 
         return self.rlc_encode(product_bytes), FQ(quotient_hi)
 
-    def rlc_to_fq_unchecked(self, word: RLC, n_bytes: int) -> Tuple[FQ, FQ]:
-        return self.bytes_to_fq(word.le_bytes[:n_bytes]), self.is_zero(
-            self.sum(word.le_bytes[n_bytes:])
```

Comment:
Why removing this? Rebase mistake maybe?

Reply:
No. I found no place uses this function and I think it'd be safer to convert `RLC` value to `FQ` with `n_bytes` constraint.

Reply:
kk fine then. Remove due to unused seems fine to me.